### ISSUE TITLE
Cannot install depext

### ISSUE THREAD

**Issue by VincentSe** [fdopen/opam-repository-mingw#45]
Hello, 

After following this page, http://fdopen.github.io/opam-repository-mingw/installation/, I tried to do
opam install depext

Then the Cygwin shell failed with this error. Is the source code really missing ?

The following actions will be performed:
  ←[32mΓêù ←[m install ←[01mdepext←[m 1.0.5

←[36m=-=-←[m ←[01mGathering sources←[m ←[36m=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=←[m
←[K←[31m[ERROR]←[m The sources of the following couldn't be obtained, aborting:
          - depext.1.0.5
        (This might be due to outdated metadata, in this case run 'opam update')

**Comment by fdopen** [comment]
No, it's not missing. It's probably just a temporary problem with github. Try it again with `opam install --debug depext`
It should print a more helpful error message, if it fails again.

**Comment by VincentSe** [comment]
The --debug switch helped : the problem was an SSL error in curl downloading opam-depext-full-1.0.5.tbz.

I'm behing a proxy and the security configuration has probably changed recently. I worked around this by manually downloading the file in firefox, then putting it in my opam local repo.

A more precise error message would have been helpful, even the --debug switch didn't tell me the SSL problem. I had to run the curl command in a shell to see it.
